frontend/src/components/pages/connect/dynamic-ui/PropertyGroup.tsx

```diff
@@ -40,33 +39,27 @@ export const PropertyGroupComponent = observer((props: {
         return (
             <div className="dynamicInputs">
                 {filteredProperties.map((p) => (
-                    <PropertyComponent key={p.name} property={p} />
+                    <PropertyComponent key={p.name} property={p}/>
                 ))}
 
-                <div style={{ gridColumn: 'span 4', paddingLeft: '8px' }}>
-                    <Collapse ghost bordered={false}>
-                        {subGroups.map((subGroup) => (
-                            <Collapse.Panel
-                                className={subGroup.propertiesWithErrors.length > 0 ? 'hasErrors' : ''}
-                                key={subGroup.group.name ?? '<null>'}
-                                header={
-                                    <div style={{ display: 'flex', alignItems: 'center', gap: '1em' }}>
-                                        <span style={{ fontSize: '1.1em', fontWeight: 600, fontFamily: 'Open Sans' }}>
+                <div style={{gridColumn: 'span 4', paddingLeft: '8px'}}>
```

Comment:
I haven't modified styles here because this code is currently not used (because of the backend) and I had no easy way to test it